spring-cloud-gcp-docs/src/main/asciidoc/spring-stream.adoc

```diff
@@ -58,10 +58,12 @@ spring.cloud.stream.gcp.pubsub.bindings.events.producer.auto-create-resources=tr
 
 If automatic resource creation is turned ON and the subscription and/or the topic do not exist for a consumer, a subscription and potentially a topic will be created.
 The topic name will be the same as the destination name, and the subscription name will be the destination name followed by the consumer group name.
-If the consumer group is not specified, an anonymous one will be created with the name `anonymous.<destinationName>.<randomUUID>`.
+
+Regardless of the automatic resource creation setting, if the consumer group is not specified, an anonymous one will be created with the name `anonymous.<destinationName>.<randomUUID>`.
```

Comment:
Regarding "automatic resource creation setting": 

I think you need some hyphens in there. Without hyphens it can mean that the setting for resource creation is itself automatic, rather than it being a manual setting that makes resource creation automatic.